react/features/speaker-stats/components/SpeakerStats.js

```diff
@@ -101,16 +112,42 @@ class SpeakerStats extends Component<Props, State> {
         return (
             <Dialog
                 cancelKey = { 'dialog.close' }
-                submitDisabled = { true }
-                titleKey = 'speakerStats.speakerStats'>
+                submitDisabled = { true } >
                 <div className = 'speaker-stats'>
+                    <SpeakerStatsHeader onSearch = { this._onSearch } />
                     <SpeakerStatsLabels />
                     { items }
                 </div>
             </Dialog>
         );
     }
 
+    /**
+     * Update the internal state with the latest speaker stats.
+     *
+     * @returns {void}
+     * @private
+     */
+    _getSpeakerStats() {
+        const stats = { ...this.props.conference.getSpeakerStats() };
+
+        if (this.state.criteria) {
+            const searchRegex = new RegExp(this.state.criteria, 'gi');
+
+            for (const id in stats) {
+                if (Object.prototype.hasOwnProperty.call(stats[id], '_isLocalStats')) {
```

Comment:
Is there a reason for using `call` here, instead of `stats[id].hasOwnProperty`?

Reply:
Yes, there should be a better way to do it than that, I will change it, thank you for your remark.

Reply:
I has been changed